src/screens/eventListScreen/EventListScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import BaseContainer from '@components/baseContainer';
+import CustomHeader from '@components/header';
+import {Button} from '@src/components/button';
```

Comment:
just in above line you have added `@components/...`, but still in next there's `@src/components`